recipes/server_os_packages.rb

```diff
@@ -21,28 +21,37 @@
 # install packages for icinga2 / classicui2 / web2
 
 case node['platform_family']
-when 'debian'
-  # package libjpeg62-dev conflicts with libgd2-xpm-dev
-  # perhaps can be removed.
-  os_packages = %w(g++ mailutils php5 php5-cli php5-fpm build-essential
+  when 'debian'
+    case node['lsb']['codename']
+      when 'trusty'
+        # package libjpeg62-dev conflicts with libgd2-xpm-dev
+        # perhaps can be removed.
+        os_packages = %w(g++ mailutils php5 php5-cli php5-fpm build-essential
                    libgd2-xpm-dev libjpeg62 libpng12-0
-                   libpng12-dev libapache2-mod-php5 imagemagick
+                   libpng12-dev libapache2-mod-php5 git-core imagemagick
```

Comment:
Probably not. That's probably a copy-paste issue. Should I look into installing that package in a different file (server_web2 perhaps?)